src/material-examples/nested-menu/nested-menu-example.ts

```diff
@@ -0,0 +1,10 @@
+import {Component} from '@angular/core';
+
+/**
+ * @title Nested menu
+ */
+@Component({
+  selector: 'nested-menu-example',
+  templateUrl: 'nested-menu-example.html',
```

Comment:
Add the CSS file here? Otherwise, people might edit later without realizing it's not linked.